todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskContract.java

```diff
@@ -41,7 +41,9 @@
 
         void createTask(String title, String description);
 
-        void updateTask( String title, String description);
+        void updateTask(String title, String description);
```

Comment:
sorry I missed this, createTask and updateTask need to be removed and made private in the concrete class.